src/chocolatey/StringResources.cs

```diff
@@ -0,0 +1,36 @@
+﻿// Copyright © 2017 - 2022 Chocolatey Software, Inc
+// Copyright © 2011 - 2017 RealDimensions Software, LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+//
+// You may obtain a copy of the License at
+//
+// 	http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+namespace chocolatey
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Linq;
+    using System.Text;
+
+    public static class StringResources
+    {
+        public static class OptionDescriptions
+        {
+            public const string DISPLAY_HEADERS = "Display headers - Display headers when limit-output is used. Requires <INSERT RELEASED VERSION HERE>";
+        }
+
+        public static class Options
+        {
+            public const string DISPLAY_HEADERS = "display-headers|use-headers";

```

Comment:
Lets not use aliases at all if we don't need to. Let's use `--use-headers`.

Reply:
Would `add-headers` / `display-headers` be more accurate than `use`? Using the headers is something code further along the theoretical pipeline will do, not Choco (to me, at least).